generated/nidcpower/nidcpower/session.py

```diff
@@ -6286,7 +6348,7 @@ def get_ext_cal_last_date_and_time(self):
         Returns the date and time of the last successful calibration.
 
         Returns:
-            month (hightime.datetime): Indicates date and time of the last calibration.
+            last_cal_datetime (hightime.datetime): Indicates date and time of the last calibration.
```

Comment:
I see this name matches `niscope`. But not `nifgen`. Can you make them all consistent?

Reply:
Yes, but that requires new metadata exports for the other drivers. I have filed an issue to address this separately: #1716